plugins/woocommerce/src/Blocks/BlockTypes/ShopperCollection.php

```
@@ -22,6 +22,67 @@ final class ShopperCollection extends AbstractBlock {
+	}
+
+	/**
+	 * Set the `listName` attribute on the auto-injected block.
+	 *
+	 * @param array|null $parsed_hooked_block The parsed hooked block array, or null to suppress insertion.
+	 * @param string     $hooked_block_type   The hooked block type name.
+	 * @param string     $relative_position   Position of the insertion point.
+	 * @param array      $parsed_anchor_block The anchor block, in parsed block array format.
+	 * @return array|null
+	 */
+	public function set_hooked_block_attributes( $parsed_hooked_block, $hooked_block_type, $relative_position, $parsed_anchor_block ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$hooked_block_type'. (undefined)

(UnusedFormalParameter)